torch/csrc/jit/runtime/profiling_graph_executor_impl.cpp

```diff
@@ -65,6 +65,21 @@ std::atomic<size_t>& getBailoutDepth() {
   return bailout_depth;
 }
 
+void removeProfilingNodes(Block* b) {
```

Comment:
This is an unrelated change, why is it in here, could you remove.

Reply:
I'll remove it. it's somewhat related, but it will be only needed when we start using fallbacks in `specialize_autogradzero.cpp`